extensions/test-processors/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@
 # under the License.
 #
 
-if (SKIP_TESTS AND NOT ENABLE_TEST_PROCESSORS)
```

Comment:
Test processors can be used in both unit and docker tests. As SKIP_TESTS is always disabled in docker builds, it should not be relevant in docker tests. In unit tests there should not be an "AND" but an "OR" logical connection between skipping the tests and enabling the test processors, but that would break the test processor build in the docker tests. Because of this we should just skip checking the SKIP_TESTS flag altogether.